docs/pages/docs/walkthrough/raw.md

```diff
@@ -53,15 +53,18 @@ err := client.Prisma.QueryRaw("SELECT * FROM `Post` WHERE id = ? AND title = ?",
 
 #### Custom Query
 
+The Prisma client doesn't support aggregations out of the box. But you can do that via a custom query:
+
 ```go
-// note the usage of db.RawString, db.RawInt, etc.
 var res []struct{
 	PostID   db.RawString `json:"post_id"`
-	Comments db.RawInt    `json:"comments"`
+	Comments db.RawInt    `json:"n_comments"`
 }
-err := client.Prisma.QueryRaw("SELECT post_id, count(*) as comments FROM `Comment` GROUP BY post_id").Exec(ctx, &res)
+err := client.Prisma.QueryRaw("SELECT post_id, count(*) as n_comments FROM `Comment` GROUP BY post_id").Exec(ctx, &res)
 ```
 
+Note that the query uses `db.RawString` etc in the struct definition to maintain compatibility. Note also that the results are an array of structs, not a struct.
```

Comment:
not sure about this, would need a mention in the postgres part as well. Maybe I need to refactor this page with some mdx components to have a language switcher in the docs as it's a lot of duplicatoin here, but haven't gotten the chance